recipes-images/images/console-hostmobility-image.bb

```diff
@@ -46,14 +46,12 @@ IMAGE_INSTALL:append:mxv-base = " \
 IMAGE_INSTALL:append:imx8mp-var-dart = " \
     libiio \
     ethtool \
-    mx5-platform-version \
 "
 
 # HMM needed tools. unclear if libiio is needed. i2c-tool could be remove later on released product.
 # TODO move and rename mx5-platform-version (package from hm-commercial) to be a machine feature in bsp layer
```

Comment:
You can remove this TODO, right?

Reply:
yes and it will be build issue in case we run with the now merge bsp branch

Reply:
also build history is part of machine configuration in my mind so it should never been herein the first place